src/ActionGroup.php

```diff
@@ -0,0 +1,76 @@
+<?php
+declare (strict_types=1);
+
+namespace LotGD\Core;
+
+/**
+ * A grouping of navigation actions, like a submenu.
+ */
+class ActionGroup
+{
+    private $id;
+    private $title;
+    private $sortKey;
+    private $actions;
+
+    /**
+     * Create a new ActionGroup, starting with an empty set of actions.
+     * @param string $id Unique identifier for this group, in the vendor/module/group format.
+     * @param string $title Title to display to the end user. Empty string means no title.
+     * @param string $sortKey Navigation menus are displayed in the order sorted by this string.
```

Comment:
Every group has a sortkey, and in the end, the groups are ordered by this string, correct? So if "Main Nav" is C and "Special Clan Nav" is A, Special Clan Nav will have priority over Main Nav?